OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -119,20 +119,23 @@ public async Task<SearchResult<WorkshopCard>> GetByFilter(WorkshopFilter filter)
         }
 
         /// <inheritdoc/>
-        public async Task<IEnumerable<WorkshopDTO>> GetByProviderId(long id)
+        public async Task<IEnumerable<WorkshopCard>> GetByProviderId(long id)
```

Comment:
I feel some code smell in naming - `GetById()` returns `WorkshopDto`, while `GetByProviderId()` returns `IEnumerable<WorkshopCard>`. Results are different - names are (almost) similar. I see that we have such situation before this PR, but I'd like to clarify shouldn't it be improved

Reply:
Well, my "vision" is that method name should reflect it's action and result, otherwise we have to check method implementation, that breaks abstraction and encapsulation principles. 

As it's only about naming - I don't insist, but my inner perfectionist is sad )